Learning-Resources.md

```diff
@@ -1,16 +1,30 @@
-### Beginners
+In general, it's easier to contribute to the Oppia codebase if you have some knowledge of git, as well as at least one of Python or AngularJS/Angular. You don't need to know all of this before you start, though! Many of our contributors have picked these skills up concurrently while tackling their first issues.
 
-The following tutorials are the bare minimum which will help you get started:
+That said, we strongly recommend that you be open to learning new things. If you need to brush up on some of the technologies used in Oppia, here are some resources that may help:
 
-- **Git**: https://learngitbranching.js.org/ -- try the following levels:
-  - Level 1,2,3 from the introduction sequence, and
-  - Level 1,2,3,4,5,6 from Push and Pull Git remotes.
-- **GitHub**: [Intro to GitHub](https://lab.github.com/courses/introduction-to-github)
+* Git and Github are used to make changes to the repository, so it's good to know how to use them to do basic stuff like branching, merging, pulling, pushing, and committing.
 
-### Advanced
+  * For beginners:
 
-These tutorials are for slightly more advanced topics, which are good to learn, but not necessary in order to make your first PR.
+    * [Learn Branching Git](https://learngitbranching.js.org/) helps explain how git works. Try the levels below:
 
-- **Git**: the other levels from https://learngitbranching.js.org/
-- **Visualizing Git**: http://git-school.github.io/visualizing-git/
-- **GitHub**: [Managing merge conflicts](https://lab.github.com/courses/managing-merge-conflicts)
+      * Levels 1, 2, and 3 from the Introduction sequence.
+      * Levels 1, 2, 3, 4, 5, and 6 from Push and Pull Git Remotes.
+
+    * [Introduction to GitHub](https://lab.github.com/courses/introduction-to-github) covers how to use GitHub.
+
+  * More advanced:
+
+    * The other levels from [Learn Branching Git](https://learngitbranching.js.org/) cover git in more depth.
+    * You may find this [git visualizer](https://git-school.github.io/visualizing-git/) helpful for understanding more advanced git operations. It can be helpful for simple ones too!
+    * GitHub's [managing merge conflicts page](https://lab.github.com/courses/managing-merge-conflicts) explains how to address merge conflicts.
+
+* AngularJS (v1) and Angular 2+ are used for Oppia's frontend; we are currently in the process of migrating from the former to the latter.  Most of our open issues are in the frontend and require at least some knowledge of HTML, AngularJS/Angular and CSS, so knowledge of frontend coding will serve you well if you'd like to contribute to Oppia over the longer term.
```

Comment:
Split AngularJS and Angular. Note that AngularJS is not for new code. Explain that we are in the middle of migrating

Reply:
done